pkgs/top-level/all-packages.nix

```diff
@@ -12220,7 +12220,9 @@ with pkgs;
   cargo-deny = callPackage ../development/tools/rust/cargo-deny {
     inherit (darwin.apple_sdk.frameworks) Security;
   };
-  cargo-embed = callPackage ../development/tools/rust/cargo-embed { };
+  cargo-embed = callPackage ../development/tools/rust/cargo-embed { 
```

Comment:
```suggestion
  cargo-embed = callPackage ../development/tools/rust/cargo-embed {
```

Reply:
Please don't hide comments and instead mark the conversation as resolved.